src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableExtensions.Minimal.cs

```diff
@@ -147,7 +147,7 @@ internal static bool TryCopyTo<T>(this IEnumerable<T> sequence, T[] array, int a
         /// <param name="count">The number of elements in the sequence.</param>
         /// <returns>The array.</returns>
         /// <remarks>
-        /// This is more efficient than the <see cref="Enumerable.ToArray{TSource}"/> extension method
+        /// This is more efficient than the <see cref="M:Enumerable.ToArray{TSource}"/> extension method
```

Comment:
❕ This is not a correctly qualified reference

Reply:
Thank you for checking reference, I will double check and update all of them.